### ISSUE TITLE
"\n" on linux asp.net core should be '\n'

### ISSUE THREAD

**Issue by themicster** [AsterNET/AsterNET#139]
In ManagerReader.cs I was having a problem where it would connect and give me a timeout waiting for protocol identifier. The problem turned out to be I just needed to change the string to a character representation of \n. Apparently there is a difference, maybe because of the encoding or the platform it's running on.

**Comment by skrusty** [comment]
Did you compile it yourself for .NET Core? Currently there's no support, however as we've now changed the primary branch to represent version 2, I would be happy at looking at any issues thrown up by a move to CORE. AS such, if you can propose a RP for compatibility between the two, i'd be happy to review it.

**Comment by themicster** [comment]
I did compile it myself. I also converted the project file to target netstandard2.0 but I doubt this was necessary. I was debugging on my windows machine and it worked before the change, but when I deployed to my server and ran it in a docker container it wouldn't work. After making the change I was still able to debug on my windows machine, so I would think this change is compatible with both platforms. I'll make a pull request.
